src/app/components/dashboard-apps-dock/dashboard-apps-dock.less

```diff
@@ -230,7 +230,7 @@ ul.dock  > li.prev img.app-logo {
 }
 
 #mobile-dock {
-  background-color: @decorator-alt-color;
+  background-color: @bg-on-bg-inverse-color;
```

Comment:
@alexnoox This variable point at the theme.less and not the variable.less.
I will change to point at an element in variable.less with the same value :)